shell/platform/fuchsia/flutter/accessibility_bridge.h

```diff
@@ -127,19 +142,25 @@ class AccessibilityBridge
   // May result in a call to FuchsiaAccessibility::Commit().
   void PruneUnreachableNodes();
 
+  // Updates the on-screen positions of accessibility elements.
+  //
+  // This should be called from Update
+  void UpdateScreenRects(int32_t node_id,
+                         SkMatrix44 parent_transform,
+                         std::unordered_set<int32_t>* visited_nodes);
+
+  void GetHitNode(int32_t node_id,
```

Comment:
A bit of documentation would be nice.

Reply:
How about `std::optional<Hit> GetHitNode(...)` for clarity?

Reply:
I'm not sure if this really makes things clearer, but did it.